### DISCUSSION TITLE
Unable to prefetch related objects when calling method ControllerBase.get_object_or_exception

### DISCUSSION THREAD

**Discussion by lewoudar** [eadwinCode/django-ninja-extra#79] [General]
From what I see in the documentation (actually this method is not really documented, I mainly look at the bookstore example demo), it is not possible to call methods [select_related](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#select-related) or [prefetch_related](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#prefetch-related).

This is a huge performance issue IMO, because we often want to fetch some related objects before using them in a template or JSON response.

**Comment by eadwinCode** [comment]
@lewoudar can you give an example

**Comment by lewoudar** [comment]
Hi @eadwinCode ,
I mean, you can't do something like the following

```python
@api_controller('/todos', tags=['todo'])
class TodoController(ControllerBase):
    @route.get('/{todo_id}', response={200: TodoSchema})
    def get_todo(self, todo_id: int):
        # select related comments with the todo object
        return self.get_object_or_exception(Todo, id=todo_id, prefecth_related=('comments',))
```

**Reply by eadwinCode** [reply]
Oh I see, 
you can do
```python
@api_controller('/todos', tags=['todo'])
class TodoController(ControllerBase):
    @route.get('/{todo_id}', response={200: TodoSchema})
    def get_todo(self, todo_id: int):
        # select related comments with the todo object
        return self.get_object_or_exception(Todo.object.prefecth_related=('comments',), id=todo_id )
```
`get_object_or_exception` takes a Django Model class or a Django QuerySet

**Reply by lewoudar** [reply]
Oh! I need to test it, thanks!
I think the methods found in `ControllerBase` should be better documented

**Reply by eadwinCode** [reply]
Yeah I will do that

**Reply by lewoudar** [reply]
thanks
